### ISSUE TITLE
Ensure monitoring for all managed apps

### ISSUE THREAD

**Issue by JosephSalisbury** [giantswarm/roadmap#601]

**Comment by JosephSalisbury** [comment]
yo @giantswarm/product-owners what up - towards improving operations of managed apps, we want to make sure we have basic monitoring in place for all our managed apps

this touches all teams, hence the gigantic umbrella issue - i've also included the kaas teams here, i have a feeling you have a few managed apps (e.g: rainbow and dex)

if you could add all the apps that you manage to the above list, and then (if necessary) create and link issues to ensure that they are scraped by prometheus, and have a basic up/down alert, that'd be wonderful <3

**Comment by JosephSalisbury** [comment]
Done for cabbage, we're gonna do linkerd when we do our next upgrade on it (linkerd is a bit complicated for monitoring :D)

**Comment by gianfranco-l** [comment]
done for honeybadger cc @kubasobon

**Comment by puja108** [comment]
@TheoBrigitte can you attach an ETA (end date) to this for the Roadmap board?

**Comment by puja108** [comment]
@gawertm I saw there's also a few unticked items from rocket, are those done or are you tracking them somewhere?

**Comment by gawertm** [comment]
@puja108 yes we are aware of them and still plan to work on them. We are tracking this ticket here as well as the linked tickets in our backlog. However building up the clusters are of higher priority right now. 

Also, I understood that Hydra (@alex-dabija) will soon start working on this one:  https://github.com/giantswarm/roadmap/issues/313 which is a general CAPI Workload Cluster monitoring. I am also hoping this can be used then for all CAPI providers and may resolve some of the issues we still have in the backlog

**Comment by TheoBrigitte** [comment]
Atlas is done here, we have monitoring in place for our managed apps. fluentbit and kibana are monitored as part of efk.

**Comment by teemow** [comment]
@TheoBrigitte can you close the issues for fluentbit and kibana then?

**Comment by TheoBrigitte** [comment]
I closed fluentbit and kibana issues, Atlas is done here, removing from our board

**Comment by JosephSalisbury** [comment]
@gawertm looks like rocket is the last with todos here, what's the status there?

**Comment by gawertm** [comment]
yes I am aware and we have it in the backlog. We will have an other look at it during next refinement to ensure nothing slipped through. Bc IIRC, it was mainly components that we do not have in production yet (or not anymore). So we would introduce monitoring for them only after we have something stable

**Comment by JosephSalisbury** [comment]
@gawertm can we kill this? it looks like it's only the capv and capvcd controllers remaining (which i somehow expect we'd have monitoring for by now)

**Comment by gawertm** [comment]
I'll check with the team on monday if they are good with the current state and then I'll close it. we def already have something in place from mario back then

**Comment by gawertm** [comment]
@JosephSalisbury not good enough for Rocket standards yet ;) we will create a separate issue specifically for that and do refinement in rocket
